Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
Yes, a lot of the problem is that those are purely the `ansible` bits and not the other 12/13 functionality additions 😢. Sorting the ansible bits are easy enough, but `poet` only takes the one argument and I don't know enough Python to file a PR against `poet` to make it take a bunch of arguments, resolve any conflicts, remove the duplicates & then sort the final list.